tests/integration/framework/process/logline/logline.go

```diff
@@ -39,6 +39,8 @@ type LogLine struct {
 	stderr            io.Reader
 	stderrExp         io.WriteCloser
 	stderrLinContains map[string]bool
+
+	outCheck chan map[string]bool
```

Comment:
```suggestion
	checkOut chan map[string]bool
```

Reply:
We can't name it this because we already have a func called this https://github.com/dapr/dapr/pull/7239/files#diff-b975baf9f073a4828870a3b544cc351547501ed77e4c82459856babb979c261dR99